modulefiles/modulefile.ProdGSI.hera

```diff
@@ -1,6 +1,12 @@
 #%Module######################################################################
-## NOAA-EMC/GSI
-##_____________________________________________________
+# NOAA-EMC/GSI
+#_____________________________________________________
+#proc ModulesHelp { } {
+#puts stderr "Set environment variables for NOAA-EMC/GSI"
+#puts stderr "This module initializes the environment "
+#puts stderr "for the Intel Compiler Suite $version\n"
+##}
+#module-whatis " NOAA-EMC/GSI whatis description"
 
 set COMP ifort
 set COMP_MP mpfort
```

Comment:
are these used anywhere?
I know CMake does not use these.

Reply:
Lines 11 and 12 removed at 2ef1c50.  Will test once Hera is returned to service.   

Note:  `COMP` and `COMP_MP` set in modulefile.ProdGSI.$machine for many machines.   Shall all occurrences of `COMP` and `COMP_MP` be removed?  What about 

`set COMP_MPI mpiifort`
`set C_COMP icc`
`set C_COMP_MP mpcc`

These remain in modulefiles.ProdGSI.hera and other machines.

Reply:
None of those are used in cmake.
If one needs to be explicit for cmake, the relevant variables are:
`CC`, `FC` and `CXX` to be `icc`, `ifort` and `icpc`

This will work so long as the `MPI_Fortran_LIBRARIES` is linked in the `target_link_libraries` for parallel applications.

If one needs to avoid the `target_link_libraries` line with `MPI_Fortran_LIBRARIES`, then one can set the above variables to `mpiicc`, `mpiifort` and `mpiicpc` respectively.
The trouble with doing that is that the serial applications will be compiled with mpi wrappers.
I don't suggest doing the latter.

Reply:
Remove `COMP`, `COMP_MP`, `COMP_MPI`, `C_COMP`, `C_COMP_MP` from `modulefiles/modulefiles.ProdGSI*`   Done at f65bac9.

Reply:
`ush/build_all_cmake.sh` runs to completion WCOSS_C, WCOSS_D, Orion, and Jet.  Can not test Hera since down for maintenance.   I do not have accounts on other platforms.